insights/parsers/hammer_compute_resource_list.py

```diff
@@ -0,0 +1,58 @@
+"""
+HammerComputeResourceList - command ``hammer --interactive 0 --output json compute-resource list``
+==================================================================================================
+
+This parser reads the compute resource list of a Satellite server using hammer in JSON format.
+It relies on the root user running the command and hammer to authenticate based on the hammer configuration file.
+If the command is unable to authenticate then no compute resources will be shown and an erro flag will be recorded in the parser
+
+Sample output from the ``hammer --interactive 0 --output json compute-resource list`` command::
+
+    [
+  {
+    "Id": 1,
+    "Name": "kvm-server",
+    "Provider": "Libvirt"
+  },
+  {
+    "Id": 4,
+    "Name": "VMWARE",
+    "Provider": "VMware"
+  },
+  {
+    "Id": 3,
+    "Name": "vmware67",
+    "Provider": "VMware"
+  }
+    ]
+
+Examples:
+
+    >>> type(hammer_compute_resources_list)
+    <class 'insights.parsers.hammer_compute_resource_list.HammerComputeResourceList'>
+"""
+
+import json
+from insights import JSONParser, parser, CommandParser
+from insights.parsers import ParseException, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.hammer_compute_resource_list)
+class HammerComputeResourceList(CommandParser, JSONParser):
+    """
+    Parse the JSON output from the ``hammer --interactive 0 --output json compute-resource list`` command.
+
+    Raises:
+        SkipException: When nothing is parsed or hammer auth fails
+
+    Attributes:
+        data(list): A list of the parsed output returned by `hammer compute resource list`
+    """
+    def parse_content(self, content):
```

Comment:
@pipopopo this method is implemented in the base `JSONParser` class so its not needed here.  You can just use `pass` here instead.